packages/lib/server/getServerErrorFromUnknown.ts

```
@@ -37,41 +38,72 @@ function parseZodErrorIssues(issues: ZodIssue[]): string {
+      data: traceId ? { ...tracedData, traceId } : undefined,
     });
   }
   if (isPrismaError(cause)) {
+    const prismaError = getServerErrorFromPrismaError(cause);
+    return new HttpError({
+      statusCode: prismaError.statusCode,
+      message: prismaError.message,
+      cause: prismaError.cause,
+      data: traceId ? { ...tracedData, traceId } : prismaError.data,
+    });
     return getServerErrorFromPrismaError(cause);
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)